tests/api/test_item_update.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-19 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test record delete."""
+import json
+
+from flask import url_for
+from invenio_accounts.models import User
+from invenio_accounts.testutils import login_user_via_session
+from invenio_circulation.proxies import current_circulation
+
+from invenio_app_ils.errors import ItemHasActiveLoanError
+
+
+def test_update_item_status(client, users, json_patch_headers,
+                            json_headers, testdata, db):
+    """Test DELETE existing location."""
+
+    def _get_item_on_loan():
+        loan_search = current_circulation.loan_search
+        for t in testdata["items"]:
+            if t['status'] == "CAN_CIRCULATE":
+                if loan_search.get_active_loan_by_item_pid(t["item_pid"])\
+                   .execute().hits.total > 0:
+                    return t
+
+    login_user_via_session(
+        client,
+        email=User.query.get(users["admin"].id).email
+    )
+    item_pid = _get_item_on_loan()['item_pid']
+    patch_op = [{"op": "replace", "path": "/status", "value": "MISSING"}]
+    url = url_for('invenio_records_rest.pitmid_item', pid_value=item_pid)
+    res = client.patch(url,
+                       headers=json_patch_headers,
+                       data=json.dumps(patch_op))
+    assert res.status_code == ItemHasActiveLoanError.code
```

Comment:
check here also message string of the exception, because all the ILSException have the same 400 code